### PR TITLE
Fpc follow standard layout

### PR DESCRIPTION
**Pull request by NaitsabesMue**

### CODE REVIEW

File: 6.3 Fast Probabilistic Consensus.md [hunk 1]
```diff
@@ -10,12 +10,23 @@ This procedure shall terminate locally if a node has not changed its opinion ove
 
 Unlike other voting-based consensus protocols, FPC uses a sequence of global random thresholds. This randomness makes FPC robust even in Byzantine environments. We refer to [FPC-BI: Fast Probabilistic Consensus within Byzantine Infrastructures](https://arxiv.org/abs/1905.10895), [Robustness and efficiency of leaderless probabilistic consensus protocols within Byzantine infrastructures](https://arxiv.org/abs/1911.08787), and [Fast Probabilistic Consensus with Weighted Votes](https://link.springer.com/chapter/10.1007/978-3-030-63089-8_24) for more details on FPC.
 
+
+The Fast Probabilistic Consensus Protocol specification depends on the following specifications:
+
++ [2.3 - Standard Payloads Layout](./2.2%20Standard%20Payloads%20Layout.md) 
++ [3.1 - Communication Models](./3.1%20Communication%20Models.md)
```
Comment (oliviasaa) [inline 6.3 Fast Probabilistic Consensus.md:17]: We don't have this file yet, are we supposed to provide the link? (honest question, I don't know...)

File: 6.3 Fast Probabilistic Consensus.md [hunk 2]
```diff
@@ -10,12 +10,23 @@ This procedure shall terminate locally if a node has not changed its opinion ove
 
 Unlike other voting-based consensus protocols, FPC uses a sequence of global random thresholds. This randomness makes FPC robust even in Byzantine environments. We refer to [FPC-BI: Fast Probabilistic Consensus within Byzantine Infrastructures](https://arxiv.org/abs/1905.10895), [Robustness and efficiency of leaderless probabilistic consensus protocols within Byzantine infrastructures](https://arxiv.org/abs/1911.08787), and [Fast Probabilistic Consensus with Weighted Votes](https://link.springer.com/chapter/10.1007/978-3-030-63089-8_24) for more details on FPC.
 
+
+The Fast Probabilistic Consensus Protocol specification depends on the following specifications:
+
++ [2.3 - Standard Payloads Layout](./2.2%20Standard%20Payloads%20Layout.md)
```
Comment (oliviasaa) [inline 6.3 Fast Probabilistic Consensus.md:16]:
```suggestion
+ [2.3 - Standard Payloads Layout](./2.3%20Standard%20Payloads%20Layout.md) 
```

**Review by oliviasaa — CHANGES_REQUESTED** [review]
there is an outdated link (section 2.3)

File: 6.3 Fast Probabilistic Consensus.md (hunk 1 above)
Comment (darcy-camargo) [reply 6.3 Fast Probabilistic Consensus.md:17]: Yeah, no problem. It saves us future work.

File: 6.3 Fast Probabilistic Consensus.md [hunk 3]
```diff
@@ -10,12 +10,23 @@ This procedure shall terminate locally if a node has not changed its opinion ove
...
+
 ## 6.3.2 FPC Protocol
 
 The FPC protocol attempts to determine consensus on the eligibility of an object `objectID`. Every node has an initial opinion `opinion` on this object. These opinions are updated in rounds until the protocol terminates using a local stopping rule. 
 
 ### 6.3.2.1 FPC parameters
-Table 5.1.1 gives a list of all parameters required for FPC.
+Table 6.3.1 gives a list of all parameters required for FPC.
 
 **Table 6.3.1 Parameters Required for FPC**
```
Comment (darcy-camargo) [inline 6.3 Fast Probabilistic Consensus.md:31]:
```suggestion
**Table 6.3.1:** Parameters Required for FPC.
```

File: 6.3 Fast Probabilistic Consensus.md [hunk 4]
```diff
@@ -35,7 +46,7 @@ Table 5.1.1 gives a list of all parameters required for FPC.
 |`MIN_MANA_PROPORTION`| double | Minimal amount of mana of received answers that allow to update opinion. If this amount is not reached the current round is not counted. 
 | `MAX_SAMPLE_SIZE`| integer | Maximal size of sample.
 
-The proposed values of the parameters are (Table 5.1.2):
+The proposed values of the parameters are (Table 6.3.2):
 
 **Table 6.3.2 FPC Parameter Default Values**
```
Comment (darcy-camargo) [inline 6.3 Fast Probabilistic Consensus.md:51]:
```suggestion
**Table 6.3.2:** FPC Parameter Default Values.
```

File: 6.3 Fast Probabilistic Consensus.md [hunk 5]
```diff
@@ -62,7 +73,7 @@ The proposed values of the parameters are (Table 5.1.2):
 Local variables are those which may be defined at the application developer's discretion and do not form a mandatory part of the protocol. They are included here to assist in understanding the protocol defined in this section. The kind of information described for these variables must be handled by the node application in some form.
 
 
-Every node shall keep the following variables (Table 5.1.3):
+Every node shall keep the following variables (Table 6.3.3):
 
 **Table 6.3.3 Required Node Variables**
```
Comment (darcy-camargo) [inline 6.3 Fast Probabilistic Consensus.md:78]:
```suggestion
**Table 6.3.3:** Required Node Variables.
```

Comment (darcy-camargo) [inline 6.3 Fast Probabilistic Consensus.md]:
```suggestion
**Table 6.3.3:** Required Node Variables.
```

File: 6.3 Fast Probabilistic Consensus.md [hunk 6]
```diff
@@ -80,7 +91,7 @@ Every node shall keep the following variables (Table 5.1.3):
 
 Note that `consensusManaList` is a list of active consensus Mana of all known nodes. As this Mana changes over time this list has to be updated over time. Such an update is expressed with the function `GetActiveConsensusMana(time)` that returns the list of active consensus Mana at time `time`.
 
-For each `objectID` that is under vote a node shall keep the following variables (Table 5.1.4):
+For each `objectID` that is under vote a node shall keep the following variables (Table 6.3.4):
 
 **Table 6.3.4 Variables Required for objectID Under Vote**
```
Comment (darcy-camargo) [inline 6.3 Fast Probabilistic Consensus.md:96]:
```suggestion
**Table 6.3.4:** Variables Required for objectID Under Vote.
```

**Review by darcy-camargo — APPROVED** [review]

**Pull request merged**